js/components/widgets/image-picker.vue

```diff
@@ -129,9 +128,9 @@ export default {
         }
     },
     methods: {
-        save: function() {
+        save() {
             if (this.HAS_FILE_API) {
-                let data = {};
+                const data = {};
```

Comment:
That's right: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const 

> in case the content is an object, this means the object itself can still be altered

👍